include/librats/conf.h

```diff
@@ -33,6 +34,8 @@ typedef struct rats_conf {
 	char attester_type[32];
 	char verifier_type[32];
 	char crypto_type[32];
+	rats_log_level_t log_level;
+	bool is_log_mode;
```

Comment:
Is it not possible to only  have `rats_log_level_t log_level` to set log level?

Reply:
Ok, then I delete is_log_mode, and judge whether the user has entered log_level by judging whether log_level is RATS_LOG_LEVEL_MAX. If the user does not enter, read the log_level in the environment parameters.